smarts/env/wrappers/gif_recorder.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from moviepy.editor import *
+import gym.envs
+import shutil
+
+
+class GifRecorder:
+    def __init__(self, dir, env):
+        self.dir = dir
```

Comment:
An improvement I could suggest is to add a timestamp or `uuid` component to the directory so there is less likelihood of collisions if there are multiple of these running.